models/getArticleByArticleIdModel.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../db/connection.js')
+
+const getArticleByArticleId = (article_id) => {
+    const articleId = Number(article_id)
+
+    if (isNaN(articleId)) {
+        return Promise.reject('invalid id')
+    } else {
+        return db.query(`
+        SELECT * FROM articles
+       WHERE article_id = ${articleId} ;
```

Comment:
Template literals are vulnerable to SQL injections - revise parameterized queries